reccmp/isledecomp/parser/node.py

```diff
@@ -21,6 +21,10 @@ def should_skip(self) -> bool:
         """The default is to compare any symbols we have"""
         return False
 
+    def is_library(self) -> bool:
+        """The default is to assume any symbols are not library functions"""
```

Comment:
```suggestion
        """The default is to assume that arbitrary symbols are not library functions"""
```
nitpick

Reply:
That does sound better. Changed it as an amended commit to keep the commit history cleaner.